tests/test_checks.py

```diff
@@ -59,36 +87,44 @@ def test_check_w2_error():
     assert result[3].startswith("The program command")
 
 
-# ===========
-# CHECK_W3
+# ===========# CHECK_W3
 # ===========
 def test_check_w3():
-    job = JobFile("w3_pass.JBI")
+      
+    program ='''SET USERFRAME
+CALL JOB:TRIGGER ARGF"PROGRAMM_EIN"\n'''
+    job_string = job_file_generator(program, foldername = "MAIN")
+    job = JobFile(job_string)
     result = check_w3(job, "W", "3")
 
     assert result is None
 
 
 def test_check_w3_error_1():
-    job = JobFile("w3_error_1.JBI")
+    program = 'CALL JOB:TRIGGER ARGF"PROGRAMM_EIN"\n'
+    job_string = job_file_generator(program, foldername = "STANDARD")
+    job = JobFile(job_string)
     result = check_w3(job, "W", "3")
 
     assert result[0] == "W"
     assert result[1] == "3"
-    assert result[2] == 6
+    assert result[2] == 5
     assert result[3] == "The command SET USERFRAME does not exist"
 
 
 def test_check_w3_error_2():
-    job = JobFile("w3_error_2.JBI")
+    program = 'CALL JOB:TRIGGER ARGF"PROGRAMM_EIN"\n'
+    job_string = job_file_generator(program, foldername = "STANDARD")
```

Comment:
```suggestion
    job_string = job_file_generator(program, foldername = "MAIN")
```
Otherwise, this looks identical to the previous test `test_check_w3_error_1()`